app/Http/Requests/OrderRequest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Http\Exceptions\HttpResponseException;
+use Illuminate\Contracts\Validation\Validator;
+
+class OrderRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'orderById' => 'string|in:true,false',
+            'fromDate' => 'date|date_format:Y-m-d',
+            'toDate' => 'date|date_format:Y-m-d',
+            'payment_status' => 'string|in:paid,unpaid',
```

Comment:
We have changed the payment_status column definition to  $table->string('payment_status')->nullable()->change();
It's not an enum of paid & unpaid anymore.

Reply:
But values for the payment status should be predefined. Like we have done in order_status (order_placed,order_rejected...).